api/src/main/java/org/openmrs/api/db/hibernate/search/LuceneAnalyzerFactory.java

```diff
@@ -58,6 +60,12 @@ public SearchMapping getSearchMapping() {
 			.filter(NGramFilterFactory.class)
 			.param("minGramSize", "2")
 			.param("maxGramSize", "20");
+		mapping.analyzerDef(LuceneAnalyzers.SOUNDEX_ANALYZER, StandardTokenizerFactory.class)
+			.filter(ClassicFilterFactory.class) //StandardTokenizerFactory https://www.programcreek.com/java-api-examples/index.php?api=org.apache.lucene.analysis.phonetic.PhoneticFilterFactory
```

Comment:
Is there any value to this comment here?

Reply:
The URL shows the example I used for the setup. I will remove it in the next commit